README_PL.md

```diff
@@ -7,9 +7,11 @@
 
 - odbiera wartości glukozy z Juggluco
 - odbiera wartości glukozy z xDrip+
+- provides several **[widgets](#widgets)** and a floating widget for the phone
```

Comment:
@froster82 : Please also update the new text in this file. Thank you!

Reply:
OK, got it. Will do.